### PR TITLE
Changes to user input, SGs, field names

### PR DESCRIPTION
**Pull request by apoorva11029**
1. Read VLAN values from accprovision Tar file

Add field 'acc_provision_tar' to inventory.yaml like this:
all:
  hosts:
    localhost:
      acc_provision_tar: /fullpath/to/tar/file

2. Modify Cisco ACI resource names to include prefix "aci-containers"

3. Add pod network security groups for pod network for DNS(workers) and ETCD(masters)

4. Neutron address-scope and pool creation
Mandatory fields required:
      aci_containers_addr_scope: addr-scp-openshiftupi
      aci_containers_l3out_vrf: uni/tn-common/ctx-sauto_l3out-1_vrf
      aci_containers_subnet_pool: subpool-openshiftupi
      aci_containers_prefix_length: 27

### CODE REVIEW

File: upi/openstack/01_security-groups.yaml [hunk 1]
```diff
@@ -149,6 +149,14 @@
       port_range_min: 2379
       port_range_max: 2380
 
+  - name: 'Create master-sg rule "etcd" for pod network'
+    os_security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      protocol: tcp
+      remote_ip_prefix: "{{ cluster_network_cidrs }}"
+      port_range_min: 2379
+      port_range_max: 2380
```
Comment (abhis2112) [inline upi/openstack/01_security-groups.yaml:158]: do we need here too 
when: os_networking_type == "CiscoACI"

File: upi/openstack/03_bootstrap.yaml [hunk 2]
```diff
@@ -25,15 +25,15 @@
     command:
       cmd: "openstack port set --tag {{ cluster_id_tag }} {{ os_port_bootstrap }}"
 
-  - name: 'Create the second bootstrap server port'
+  - name: 'Create the ACI containers bootstrap server port'
```
Comment (abhis2112) [inline upi/openstack/03_bootstrap.yaml:28]: Should be something like 'Create the bootstrap server opflex base port'
@snaiksat  can also confirm on this

File: upi/openstack/03_bootstrap.yaml [hunk 3]
```diff
@@ -25,15 +25,15 @@
...
     os_port:
-      name: "{{ os_port_bootstrap2 }}"
-      network: "{{ os_network2 }}"
+      name: "{{ os_aci_containers_port_bootstrap }}"
+      network: "{{ os_aci_containers_network }}"
     when: os_networking_type == "CiscoACI"
 
-  - name: 'Set second bootstrap port tag'
+  - name: 'Set ACI containers bootstrap port tag'
```
Comment (abhis2112) [inline upi/openstack/03_bootstrap.yaml:34]: Same here

File: upi/openstack/03_bootstrap.yaml [hunk 4]
```diff
@@ -25,15 +25,15 @@
     command:
       cmd: "openstack port set --tag {{ cluster_id_tag }} {{ os_port_bootstrap }}"
 
-  - name: 'Create the second bootstrap server port'
+  - name: 'Create the ACI containers bootstrap server port'
     os_port:
-      name: "{{ os_port_bootstrap2 }}"
-      network: "{{ os_network2 }}"
+      name: "{{ os_aci_containers_port_bootstrap }}"
```
Comment (abhis2112) [inline upi/openstack/03_bootstrap.yaml:30]: Could be os_opflex_base_port_bootstrap

File: upi/openstack/03_bootstrap.yaml [hunk 5]
```diff
@@ -25,15 +25,15 @@
...
+      network: "{{ os_aci_containers_network }}"
```
Comment (abhis2112) [inline upi/openstack/03_bootstrap.yaml:31]: os_opflex_network

File: upi/openstack/04_control-plane.yaml [hunk 6]
```diff
@@ -29,18 +29,18 @@
       cmd: "openstack port set --tag {{ cluster_id_tag }} {{ item.1 }}-{{ item.0 }}"
     with_indexed_items: "{{ [os_port_master] * os_cp_nodes_number }}"
 
-  - name: 'Create the second Control Plane ports'
+  - name: 'Create the ACI containers Control Plane ports'
```
Comment (abhis2112) [inline upi/openstack/04_control-plane.yaml:32]: Similarly here based on previous comments and for the following changes

File: upi/openstack/03_bootstrap.yaml (hunk 2 above)
Comment (snaiksat) [reply upi/openstack/03_bootstrap.yaml:28]: this one is fine (i was using opflex terminology before but i changed it)

File: upi/openstack/03_bootstrap.yaml (hunk 3 above)
Comment (snaiksat) [reply upi/openstack/03_bootstrap.yaml:34]: this one is fine (i was using opflex terminology before but i changed it)

File: upi/openstack/03_bootstrap.yaml (hunk 4 above)
Comment (snaiksat) [reply upi/openstack/03_bootstrap.yaml:30]: this one is fine (i was using opflex terminology before but i changed it)

File: upi/openstack/03_bootstrap.yaml (hunk 5 above)
Comment (snaiksat) [reply upi/openstack/03_bootstrap.yaml:31]: this one is fine (i was using opflex terminology before but i changed it)

File: upi/openstack/04_control-plane.yaml (hunk 6 above)
Comment (snaiksat) [reply upi/openstack/04_control-plane.yaml:32]: this one is fine (i was using opflex terminology before but i changed it)

File: upi/openstack/01_security-groups.yaml (hunk 1 above)
Comment (snaiksat) [reply upi/openstack/01_security-groups.yaml:158]: we can do this as a follow up, but lets do that after we get confirmation from RH that its actually not required.

**Pull request merged**